README/ReleaseNotes/v636/index.md

```diff
@@ -16,7 +16,7 @@ The following people have contributed to this new version:
  Jakob Blomer, CERN/EP-SFT,\
  Philippe Canal, FNAL,\
  Mattias Ellert, Uppsala University,\
- Florine de Geus, CERN/University of Twente,\
+ Florine de Geus, CERN/EP-SFT and University of Twente,\
  Fernando Hueso Gonzalez, CSIC/University of Valencia,\
```

Comment:
Hm, I'm missing entirely :scream: maybe you can add it if you push before merging, otherwise I'll open a PR afterwards